### DISCUSSION TITLE
EE680DCF-96C1-ABA0-51B4-07029A79D696

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#144] [General]
# EE680DCF-96C1-ABA0-51B4-07029A79D696

The Sombrero Galaxy (also known as Messier 104 or NGC 4594) is an Sa-type spiral galaxy about 31.1 million light-years away in the constellation Virgo. It is 49,000 light-years across and is one of the brightest galaxies seen through telescopes

(NOTE: Disable 

https://celestia.mobi/addon?item=EE680DCF-96C1-ABA0-51B4-07029A79D696&lang=es

**Comment by Pascualito1008** [comment]
sombrero is not a hat
